pkg/nodeinfo/filter.go

```diff
@@ -72,6 +75,13 @@ func (b *NodeLabelFilterBuilder) WithLabel(key, val string) *NodeLabelFilterBuil
 	return b
 }
 
+// WithLabelIgnoreValue adds a label for the Build process of the Label filter
+// use special constant as value, which mean that nodeLabelFilter will ignore value when processing nodeLabel
+func (b *NodeLabelFilterBuilder) WithLabelIgnoreValue(key string) *NodeLabelFilterBuilder {
```

Comment:
`GetNodeAttributes()` accepts multiple filters.

you can define a new filter `nodeLabelNoValFilter` which will filter according to label presence without the value.

then you would not change logic to existing filter and avoid using special value.

Reply:
Done. I moved this logic to separate `nodeLabelNoValFilter` filter

Reply:
Thanks!